students/sangwoong/users/views.py

```diff
@@ -1,3 +1,47 @@
 from django.shortcuts import render
 
-# Create your views here.
+import json, re
```

Comment:
1~2번째에 사용 안 하는 package는 제거해 주세요.
```suggestion
import json
import re
```